app/tpl/show.html

```diff
@@ -57,6 +57,7 @@
                 <span style='text-transform:uppercase;font-size:80%;padding-left:20px;color:#{{ item.color }}'>
                     <img id='bodyToggleImg_{{ item.id }}' data-type='{{ Ev.Entity.type }}' class='bodyToggleImg' src='app/img/show-more.png' title='Show more' alt='+' /> {{ item.category }}</span>
                 <span class='date'><img class='image' src='app/img/calendar.png' />{{ item.date|kdate }}</span>
+                <span class="experimentstepicon" onclick='es.open( {{ item['id']}} )'><img class='image' src='app/img/status.png' /></span>
```

Comment:
all the onclick and inline javascript is to be removed. Please rewrite without inline JS. This is to be able to put a restrictive content security policy header once all the inline JS is removed.